### ISSUE TITLE
[Redesign]: Deviations from mockup

### ISSUE THREAD

**Issue by fllp** [protz/thunderbird-conversations#1172]
This may be nit-picking but i noticed some differences in the current implentations of the redesign. Mostly the deviations were introduced with #1165. How should they be treated? Personally, i'd prefer the mockup versions.  
Comparisions below. @protz, @xi and @elioquoshi: What's your opinion

### Duplicate reply all action

- applies to reply to list as well
- should at least be an icon instead of text link   
- IMHO the `|` as separator doesn't fit into the new design

Original mockup:  
![mockup-reply-all](https://cloud.githubusercontent.com/assets/15898292/24366084/52a7f632-1318-11e7-83cc-e113d5aedf74.png)

Current state:  
![reply-all](https://cloud.githubusercontent.com/assets/15898292/24365987/0a1d6eec-1318-11e7-9d6f-5ba54c078f55.png)

### Attachment icons

I just dont like the floppy icon :stuck_out_tongue:

Original mockup:
![mockup-attachments](https://cloud.githubusercontent.com/assets/15898292/24366477/862c4d72-1319-11e7-8080-efd7bbd695ea.png)

Current state:
![attachments](https://cloud.githubusercontent.com/assets/15898292/24366464/7e862d54-1319-11e7-95f1-30db4a494168.png)

**Comment by protz** [comment]
Ok to replace floppy with downarrow.

The reply all link at the top is important, because sometimes emails are
super long and you don't want to scroll all the way to the bottom. Can you
make it an icon?

Thanks!

Jonathan

**Comment by protz** [comment]
Also, the (i) info icon seems a little too big now...? I'm wondering if
that's something we can fix

**Comment by protz** [comment]
Can we also replace the "preview" icon with an eye? I'd think this is the standard idiom for preview icons.

**Comment by xi** [comment]
@fllp thanks for the summary!

> The reply all link at the top is important, because sometimes emails are super long and you don't want to scroll all the way to the bottom. Can you make it an icon?

+1

> Ok to replace floppy with downarrow.

For me the downarrow is confusing. I guess it is carried over from webservices like gmail, where you need to download the attachment. But in thunderbird, the download metaphor does not make a lot of sense.

The floppy on the other hand fitting and on top of the much more specific than an arrow.

> Can we also replace the "preview" icon with an eye? I'd think this is the standard idiom for preview icons.

See https://github.com/protz/thunderbird-conversations/issues/1164#issuecomment-286216396. (I think I closed that issue by accident btw)

**Comment by protz** [comment]
Thanks! I reopened the issue

**Comment by protz** [comment]
So, I compared before and after... before the merge:

![image](https://cloud.githubusercontent.com/assets/10673049/24442928/3431af74-1415-11e7-9d65-25af4300d92e.png)
(good size)

![image](https://cloud.githubusercontent.com/assets/10673049/24442935/4371c47e-1415-11e7-8e7d-3e1522516acb.png)

(good icons, lack of icon for preview, PDF Icon not centered, PDF Icon clickable)

after the merge:

![image](https://cloud.githubusercontent.com/assets/10673049/24442953/64d03394-1415-11e7-8f53-988c4f9a03a6.png)
(PDF Icon centered but *not clickable anymore*, icons too big, surprising floppy icon)

![image](https://cloud.githubusercontent.com/assets/10673049/24442969/78f3d0c4-1415-11e7-971d-298a49df10d6.png)
(much bigger icons)

- can we restore icons to their normal size
- get back to the previous icon for download
- make the PDF icon clickable again
?

Thanks,

Jonathan

**Comment by xi** [comment]
> can we restore icons to their normal size

For me, the icons are very small and hard to click, that's why I made them bigger. Maybe it depends on some OS settings…

> get back to the previous icon for download

What about https://github.com/protz/thunderbird-conversations/issues/1172#issuecomment-289581367 ?

> make the PDF icon clickable again

I created #1178 for that.

**Comment by elioqoshi** [comment]
Agreed with what @protz said. Once the issues are fixed let me know if you need any help

**Comment by fllp** [comment]
> For me, the icons are very small and hard to click, that's why I made
> them bigger. Maybe it depends on some OS settings…

Do you  happen to have a 4k display? Then this is an issue of
thunderbrid itself. On HiDPI displays, firefox and thunderbird are known
to have issues. You'd have to change the display scaling manually in the
thunderbird config.

**Comment by xi** [comment]
> Do you  happen to have a 4k display?

nope.

Here is a screenshot

![screenshot from 2017-03-29 21 24 56](https://cloud.githubusercontent.com/assets/202576/24472506/3e02c0d6-14c6-11e7-96e0-ae65bd7c6025.png)

**Comment by protz** [comment]
I get the feeling that you adjusted the size of the icons based on your custom font/color/OS theme, which may not be representative of the general population...?

**Comment by xi** [comment]
I am not sure if I understand the issue correctly. The relevant change is 5a35526b7bea0cad245717e2adff7c2cda327301. As you can see, the size of the icons is defined relatively to font size. I posted a screenshot of the current master (9ee33d6) on my setup in https://github.com/protz/thunderbird-conversations/issues/1172#issuecomment-290198573. What is the actual situation?

- The relation of font size to icon size in the screenshot is correct, but it is different with other setups.
  - In this case, we need to find a solution that works with all or at least most setups.
- The relation of font size to icon size in the screenshot is not correct and should be changed.
  - In this case, I have argued that smaller icons are hard to click (the current size is already much smaller than the link was before). I would like to discuss that argument in order to make an informed decision.
- The icon size should not be relative to font size in the first place.
  - In that case, how should the icon size be defined instead?

**Comment by Standard8** [comment]
Closing this out, as this is very old now, and we probably aren't going to do any additional changes specifically for this.
